server/member/lease.go

```diff
@@ -101,7 +101,7 @@ func (l *lease) KeepAlive(ctx context.Context) {
 	wg := sync.WaitGroup{}
 	wg.Add(1)
 	go func() {
-		l.renewLeaseBg(ctx1, l.timeout/3, renewed)
+		l.renewLeaseBg(ctx1, renewed)
```

Comment:
```suggestion
	        // keepAliveInterval is set internally by `ETCD` client, we don't need to set it explicitly. 
                // nextKeepAlive := time.Now().Add((time.Duration(karesp.TTL) * time.Second) / 3.0)
		l.renewLeaseBg(ctx1, renewed)
```
Maybe we need add some comment to explain how the `keepAliveInterval` is added to `keepAliveSteam`.